posthog/models/web_preaggregated/sql.py

```diff
@@ -0,0 +1,295 @@
+from django.conf import settings
+
+CLICKHOUSE_CLUSTER = settings.CLICKHOUSE_CLUSTER
+CLICKHOUSE_DATABASE = settings.CLICKHOUSE_DATABASE
+
+
+def TABLE_TEMPLATE(table_name, columns, order_by, on_cluster=True):
```

Comment:
I assume you've tested this extensively in prod already?

Reply:
Oh yes :)